lib/round.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/deck'
+
+
+class Round
+
+attr_reader :deck, :turns, :number_correct
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+  end
+
+  def current_card
+    deck.cards.first
+  end
+
+  def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+      @turns << new_turn
+      if new_turn.correct?
+        @number_correct += 1
+      end
+    @deck.cards.shift
+    new_turn
+  end
+
+  def number_correct_by_category(card_category)
+    category_count = 0.0
+    @turns.each do |turn|
+      if turn.card.category == card_category && turn.correct?
+        category_count += 1
+      end
+    end
+    category_count
+  end
+
+  def percent_correct
+    (number_correct / turns.count) * 100
```

Comment:
If you can call to the instance variable instead of the attr_Reader, try to remember to do that. When you call to `number_correct`, you're calling to the attr_reader, which returns the instance variable. But you can call directly to the instance variable and cut out the middle man instead.